shared/src/commonMain/sqldelight/db/chargehub/CarDb.sq

```diff
@@ -1,6 +1,6 @@
 CREATE TABLE Car (
     id INTEGER PRIMARY KEY,
-    userId INTEGER,
+    userId INTEGER NOT NULL,
```

Comment:
Just now that i see this, our api's use strings for id's for 90% of the time

Reply:
Alright, I will change this